From a6b818693ccfa2fa3298b0f8d900959acb24bb6d Mon Sep 17 00:00:00 2001
From: Francisco Boni Neto <boboniboni@gmail.com>
Date: Wed, 9 Feb 2022 10:50:23 -0300
Subject: [PATCH] Tell the truth about private API

---
 mkspecs/features/qt_module.prf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mkspecs/features/qt_module.prf b/mkspecs/features/qt_module.prf
index 6ef7dbc6..fe572cc2 100644
--- a/mkspecs/features/qt_module.prf
+++ b/mkspecs/features/qt_module.prf
@@ -218,9 +218,9 @@ android: CONFIG += qt_android_deps no_linker_version_script
     QMAKE_LFLAGS += $${QMAKE_LFLAGS_VERSION_SCRIPT}$$verscript
 
     internal_module {
-        verscript_content = "Qt_$${QT_MAJOR_VERSION}_PRIVATE_API { *; };"
+        verscript_content = "Qt_$${QT_MAJOR_VERSION}.$${QT_MINOR_VERSION}.$${QT_PATCH_VERSION}_PRIVATE_API { *; };"
     } else {
-        verscript_content = "Qt_$${QT_MAJOR_VERSION}_PRIVATE_API {" \
+        verscript_content = "Qt_$${QT_MAJOR_VERSION}.$${QT_MINOR_VERSION}.$${QT_PATCH_VERSION}_PRIVATE_API {" \
                             "    qt_private_api_tag*;"
 
         private_api_headers = $$SYNCQT.PRIVATE_HEADER_FILES $$SYNCQT.QPA_HEADER_FILES
-- 
2.35.0

